sink/src/main/java/com/spredfast/kafka/connect/s3/sink/S3SinkTask.java

```diff
@@ -107,6 +123,39 @@ public void stop() throws ConnectException {
     }
   }
 
+  @Override
+  public Map<TopicPartition, OffsetAndMetadata> preCommit(
+      Map<TopicPartition, OffsetAndMetadata> currentOffsets) {
+    Map<TopicPartition, OffsetAndMetadata> result = new HashMap<>();
+
+    log.info("{} performing preCommit", name());
```

Comment:
I had originally had these log statements set to INFO to make debugging easier. Perhaps these should be set to DEBUG level?

Reply:
I believe the same applies to all other newly introduced INFO messages. Besides that, if we want to keep them, it would be nice to have them worded as full sentences.